### ISSUE TITLE
StateCode, what does that mean?

### ISSUE THREAD

**Issue by stefxx** [sampsyo/wideq#97]
Quick question around the "StateCode" that is being returned by the device monitoring. I noticed the following values:

S (normal), F (data isn't being refreshed anymore, related to token refresh?), and also N, P, W and E. Does anyone knows what they mean?

Thanks!

**Comment by sampsyo** [comment]
Yeah, I've puzzled over those for a while as well, mostly to no avail. Let us know if you do have any great insight into what those state codes might mean—maybe we can add an enum or something to document the states that we figure out.

**Comment by stefxx** [comment]
This is what I have seen so far:
StateCode S = monitoring OK (but might still be missing returnData)
StateCode W or P = Often after monitoring starts, just retry
StateCode F = error, requires monitoring restart
StateCode E = ???
StateCode N = ???
